sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -6,6 +6,7 @@ Azure Cognitive Services Text Analytics is a cloud service that provides advance
 * Named Entity Recognition
 * Personally Identifiable Information (PII) Recognition
 * Linked Entity Recognition
+* Healthcare Recognition
```

Comment:
just to make sure, so the decision was that it was ok to document the 3.1-preview.3 features even though they are in private preview?